apps/nextjs/src/components/board/mobile/mobile-board.module.css

```
@@ -0,0 +1,87 @@
+  padding-bottom: 10px;
+  overflow-x: clip;
+}
+
+.boardTitle {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)